### ISSUE TITLE
Vaughnlive stopped working. This fix worked for me.

### ISSUE THREAD

**Issue by techmouse** [streamlink/streamlink#1747]
### Checklist

- [x] This is a bug report.
- [ ] This is a feature request.
- [x] This is a plugin (improvement) request.
- [ ] I have read the contribution guidelines.

---

Vaughnlive stopped working again.

It seems all you have to do is change:
```
    rtmp_server_map = {
        "594140c69edad": "192.240.105.171:1935",
        "585c4cab1bef1": "192.240.105.171:1935",
        "5940d648b3929": "192.240.105.171:1935",
        "5941854b39bc4": "192.240.105.171:1935"
```

to:
```
    rtmp_server_map = {
        "594140c69edad": "192.240.105.171:2935",
        "585c4cab1bef1": "192.240.105.171:2935",
        "5940d648b3929": "192.240.105.171:2935",
        "5941854b39bc4": "192.240.105.171:2935"
```

**Comment by beardypig** [comment]
Dup of #1730, fixed in #1731.

Thanks for report @techmouse, this has already been noted a fixed :)

**Comment by techmouse** [comment]
@beardypig Oh my bad. I looked around in the issues section and didn't see any mention of it, so I tried to fix it myself before posting anything. I didn't know somebody already beat me to it.

I guess I'll just go ahead and ask this here before making a new thread. The ustream.tv plugin is pretty much foobar'd. It's been buggy for a while but several retries could eventually get it working in the past. But now it's not even getting audio when it works. Is there any news on the plugin?
